docs/using/installing/index.rst

```diff
@@ -340,13 +340,15 @@ The file ``cluster.key`` should be kept only by the cluster administrator; it do
 
 You are now able to generate authentication certificates for the control service and each of your nodes.
 To generate the control service certificate, run the following command from the same directory containing your authority certificate generated in the previous step.
-Replace ``example.org`` with the hostname of your control service node; this hostname should match the one you will give to REST API clients.
+Replace ``example.org`` with the hostname of your control service node; this hostname should match the hostname you will give to REST API clients.
```

Comment:
This is pre-existing ... but ... Are we really calling this the "REST API"?  It's an HTTP API.